src/Fixer/Basic/Psr0Fixer.php

```diff
@@ -92,7 +99,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $path = str_replace('\\', '/', $file->getRealPath());
             $dir = dirname($path);
 
-            if (isset($this->configuration['dir'])) {
+            if ($this->configuration['dir']) {
```

Comment:
`0` is not string and would not pass option resolving
`"0"` would pass and it's config issue. anyway, here we shall have valid values, any validation shall be done during config resolving